src/main/java/Mars.java

```diff
@@ -0,0 +1,205 @@
+import java.util.Scanner;
+import java.util.List;
+import java.util.ArrayList;
+
+
+public class Mars {
+    public static void main(String[] args) {
+        List<Task> arrayList = new ArrayList<>();
```

Comment:
```suggestion
        List<Task> taskList = new ArrayList<>();
```
The variable name arrayList could be improved to better describe its purpose. Since this is a collection of tasks, consider renaming it to tasks or taskList.